tests/check/check_signed_video_auth.c

```diff
@@ -693,8 +693,8 @@ START_TEST(remove_the_g_nalu)
   // authenticity is NOT OK.
   signed_video_accumulated_validation_t final_validation = {
       SV_AUTH_RESULT_NOT_OK, false, 17, 16, 1, SV_PUBKEY_VALIDATION_NOT_FEASIBLE, true, 0, 0};
-  struct validation_stats expected = {.valid_gops = 3,
-      .invalid_gops = 2,
+  struct validation_stats expected = {.valid_gops = 4,
```

Comment:
Can you elaborate on why you get more valid GOPs and update the description above, since that indicate there are only 3 valid GOPs.